features/utilities/pom.xml

```diff
@@ -252,6 +252,13 @@
 			<artifactId>gt-epsg-hsql</artifactId>
 			<version>${org.geotools.version}</version>
 		</dependency>
+        <dependency>
+            <groupId>org.hsqldb</groupId>
+            <artifactId>hsqldb</artifactId>
+            <version>${geotools-hsql.version}</version>
+            <type>jar</type>
+            <classifier>jdk8</classifier>
```

Comment:
jdk8?  is that correct?

Reply:
From thttps://hsqldb.org/web/hsqlFAQ.html
> A second Jar, hsqldb-jdk8.jar works with JRE 8 and later.